src/System Application/App/Performance Profiler/src/PerformanceProfilerInstaller.Codeunit.al

```diff
@@ -0,0 +1,66 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Tooling;
+
+using System.Environment.Configuration;
+using System.DataAdministration;
+using System.PerformanceProfile;
+using System.Reflection;
+using System.Upgrade;
+
+codeunit 1933 "Performance Profiler Installer"
+{
+    Subtype = Install;
```

Comment:
Check if the brand new deployment has this setup, the deployment code is tricky. See the comment below regarding upgrade.

Reply:
See answer above, we are using an existing pattern.